src/components/Header/Header.tsx

```diff
@@ -4,11 +4,18 @@ import { ReactComponent as Loupe } from "../../assets/loupe.svg";
 import MenuItems from "../MenuItems/MenuItems";
 import { MENU_ITEMS } from "./Header.constant";
 
-function Header() {
+function Header(props: { setModalActive: (arg0: boolean) => void }) {
```

Comment:
please write interface for this instead of the cheating

Reply:
done